beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryTest.java

```diff
@@ -115,7 +119,8 @@ private ExecutionPayload getExecutionPayload(final BeaconBlock block) {
     return BeaconBlockBodyBellatrix.required(block.getBody()).getExecutionPayload();
   }
 
-  private BeaconBlock assertBlockCreated(final int blockSlot, final Spec spec)
+  private BeaconBlock assertBlockCreated(
```

Comment:
We probably should add a test for the cases where `isMevBoostEnabled` and `blinded` are true (and the various combinations).  Looks like we're always passing `false, false` at the moment.

Reply:
quite not there yet. I'll do a Part-2 of the PR.